doc_source/docs/installation.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Installation
+summary: How to install ImageJ and the AMPT plugin
+authors:
+    - Patrick Johnson
+date: 2022-01-11
+---
+Installation
+==========
+
+This tutorial explains how to install ImageJ and the AMPT plugin.
+
+## Download Fiji
+
+Download and unpack Fiji using the distribution appropriate for your platform from the [Fiji
+Download](https://imagej.net/software/fiji/downloads) site.
+
+Launch Fiji.
+
+## Install AMPT
+
+Note that these instructions are a specific instance of the more generalized [Following an Update
+Site](https://imagej.net/update-sites/following) instructions from ImageJ.
+
+### Start the Updater.
+
+Start the ImageJ Updater (_Help_ > _Update..._)
+
+![Start ImageJ Updater](img/start_imagej_updater.png)
+
+### Add a new update site
+
+Click on the _Manage update sites_ button to bring up the site management dialoag.
+
+Add AMPT as a new update site.
```

Comment:
We may want to rephrase this as "If AMPT is not in the list of update sites,  add the AMPT update site"...

Reply:
Fair point. Updated in 9c1cf57.